packages/manager/src/featureFlags.ts

```diff
@@ -151,6 +151,7 @@ interface AclpAlerting {
   alertDefinitions: boolean;
   beta: boolean;
   editDisabledStatuses?: AlertStatusType[];
+  maxEmailNotificationChannelRecipients?: number;
```

Comment:
```suggestion
  maxEmailChannelRecipients?: number;
```

we can go for a small name also, just NIT